modules/smartadserverBidAdapter.js

```diff
@@ -31,15 +31,12 @@ export const spec = {
     // to find which one the ad server needs
 
     // pull requested transaction ID from bidderRequest.bids[].transactionId
-    if (!Array.isArray(validBidRequests)) {
-      validBidRequests = [validBidRequests];
-    }
     var bid = validBidRequests[0];
     const payload = {
       siteid: bid.params.siteId,
       pageid: bid.params.pageId,
       formatid: bid.params.formatId,
-      currencyCode: config.getConfig('currency'),
+      currencyCode: config.getConfig('currency').adServerCurrency,
```

Comment:
You should use `config.getConfig('currency.adServerCurency')` since the config module does safe deep access.  The way you have it here if currency hasn't been set then it will throw "can't read adServerCurrency of undefined" error.